### PR TITLE
Feat customizable commands

### PR DESCRIPTION
**Pull request by MaxwelMazur**
- customizable command for all points

### CODE REVIEW

File: pkg/cmd/webapp/build/build.go [hunk 1]
```diff
@@ -193,9 +193,13 @@ func checkArgsJson(cmd *BuildCmd) error {
 }
 
 func runCommand(cmd *BuildCmd, conf *contracts.AzionApplicationConfig, envs []string) error {
+    var command string = conf.BuildData.Cmd
+	if conf.BuildData.Cmd == "" {
+        command = conf.BuildData.Default
```
Comment (PatrickMenoti) [inline pkg/cmd/webapp/build/build.go:198]: default should always run 
cmd is the custom command

**Review by PatrickMenoti — CHANGES_REQUESTED** [review]

File: pkg/cmd/webapp/publish/publish.go [hunk 2]
```diff
@@ -408,6 +409,7 @@ func runCommand(cmd *publishCmd, conf *contracts.AzionApplicationConfig, envs []
 		fmt.Fprintf(cmd.Io.Out, "$ %s\n", command)
 
 		output, _, err := cmd.CommandRunner(command, envs)
+        fmt.Println(">>> err: ", err)
```
Comment (PatrickMenoti) [inline pkg/cmd/webapp/publish/publish.go:412]: I think you forgot to remove this

**Review by PatrickMenoti — CHANGES_REQUESTED** [review]

File: pkg/cmd/webapp/publish/publish.go (hunk 2 above)
Comment (MaxwelMazur) [reply pkg/cmd/webapp/publish/publish.go]: removed

**Review by PatrickMenoti — APPROVED** [review]
lgtm

**Review by pauloSF0 — APPROVED** [review]

**Pull request merged**